compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/platform/Constants.uikit.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2024 The Android Open Source Project
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package androidx.compose.ui.platform
+
+/**
+ * iOS default value in scale-independent points for the pan gesture slop.
+ */
+internal const val CUPERTINO_PAN_GESTURE_SLOP_VALUE = 10
```

Comment:
Let's make it typed and align naming (it's used not only in "pan" gestures)

```suggestion
internal const val DefaultCupertinoTouchSlop = 10.dp
```

Reply:
No DP here, new logic uses CGFloat(aka Double)

Reply:
Just get raw value from dp there - we need to keep such things typed

Reply:
It doesn't work well here.
Dp is not primitive - turns into just `val` instead of `const val`. 
Let's leave it as it is. Renamed to `CUPERTINO_TOUCH_SLOP`. It's not `Default`, it's what it is.